scripts/release.ts

```diff
@@ -224,7 +229,6 @@ const createRelease = async () => {
 }
 
 const mergeRelease = async () => {
-  await fetch()
```

Comment:
I think we should still keep this one, see review comment 🙏🏽

Reply:
It's not clear to me why we'd want this.

In the "merge release" path we'd end up running:

```ts
await fetch()
// then `isReleaseInProgress()` would return true, and we'd immediately
await fetch()
// again
```